_includes/footer.html

```diff
@@ -40,13 +40,15 @@
         </a>
         <a href="https://scholarslab.lib.virginia.edu/feed.xml">
           <!-- <div>RSS icon made by <a href="https://www.freepik.com/" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/"                 title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/"                 title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div> -->
-          <img src="{{ '/assets/img/svg/rss.svg' | relative_url }}">
+          <img src="{{ '/assets/img/svg/rss.svg' | relative_url }}" alt="RSS feed icon" />
```

Comment:
This is OK, and certainly better than not having at attribute at all, but it might be preferable to just include an empty `alt` attribute, so assistive tech simply skips mention of the image itself. The 'RSS feed' text for the link is sufficient, so an empty attribute would [avoid redundancy](https://webaim.org/techniques/alttext/).

Reply:
Hi @jeremyboggs I see what you're saying ... as I considered the icon as 'functional/content' and not just 'presentational/design', with it relating to the link it's wrapped in. And the aXe checker flagged it too, I thought it best to have the alt tag populated, than empty.